developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst

```diff
@@ -159,11 +159,13 @@ You'll have to install the **new** Mongo PHP extension_ and enable it:
     $ sudo phpenmod mongodb
 
 .. _adapter: https://github.com/alcaeus/mongo-php-adapter
+.. _composer: https://getcomposer.org
 
-Finally, you have to install the Mongo PHP adapter_:
+Finally, you have to install the Mongo PHP adapter_ using the dependency manager composer_:
 
 .. code-block:: bash
 
+    $ sudo apt-get install composer
```

Comment:
https://getcomposer.org/download/
The problem is not that it does not work, but that your composer is old, that's all.